src/platform/packages/shared/presentation/presentation_publishing/state_manager/types.ts

```diff
@@ -35,11 +35,11 @@ export type CustomComparators<StateType> = {
   [KeyType in keyof StateType]?: ComparatorFunction<StateType, KeyType>;
 };
 
-type SubjectsOf<T extends object> = {
+export type SubjectsOf<T extends object> = {
```

Comment:
If we undo the changes to `src/platform/plugins/shared/controls/public/controls/data_controls/options_list_control/types.ts` - we should also undo these.